src/Groups/cosets.jl

```diff
@@ -149,14 +209,74 @@ representative(C::GroupCoset) = C.repr
 """
     is_bicoset(C::GroupCoset)
 
-Return whether `C` is simultaneously a right coset and a left coset for the same subgroup `H`.
+Return whether `C` is simultaneously a right coset and a left coset for the same subgroup `H`.  This 
+is the case if and only if the coset representative normalizes the acting domain subgroup.
+
+# Examples
+```jldoctest
+julia> G = symmetric_group(5)
+Sym( [ 1 .. 5 ] )
+
+julia> H = symmetric_group(4)
+Sym( [ 1 .. 4 ] )
+
+julia> g = perm(G,[3,4,1,5,2])
+(1,3)(2,4,5)
+
+julia> gH = left_coset(H,g)
+Left coset   (1,3)(2,4,5) * Sym( [ 1 .. 4 ] )
+
+julia> is_bicoset(gH)
+false
+
+julia> f = perm(G,[2,1,4,3,5])
+(1,2)(3,4)
+
+julia> fH = left_coset(H,f)
+Left coset   (1,2)(3,4) * Sym( [ 1 .. 4 ] )
+
+julia> is_bicoset(fH)
+true
+```
 """
 is_bicoset(C::GroupCoset) = GAPWrap.IsBiCoset(C.X)
 
 """
     right_cosets(G::Group, H::Group)
 
 Return the vector of the right cosets of `H` in `G`.
+
+# Examples
+```jldoctest
+julia> G=symmetric_group(5)
+Sym( [ 1 .. 5 ] )
+
+julia> H = symmetric_group(3)
+Sym( [ 1 .. 3 ] )
+
+julia> right_cosets(G,H)
+20-element Vector{GroupCoset{PermGroup, PermGroupElem}}:
```

Comment:
Perhaps a shorter example would do, too? The user has to scroll a lot with long examples